### ISSUE TITLE
Keyboard/accessibility support

### ISSUE THREAD

**Issue by femi-zz** [john-doherty/long-press-event#30]
Hi, 

I'm using this library to support a cross platform application. While long press works great with mouse and touch, I was wondering if there was a way to make it also work with keyboard? So for example, long pressing the 'Enter' key when focused on a button would trigger the same event?

**Comment by john-doherty** [comment]
I'm not sure that makes sense unless the item you are attempting to capture is able to receive tabbed focus?

**Comment by thesimplethingsinc** [comment]
> I'm not sure that makes sense unless the item you are attempting to capture is able to receive tabbed focus?

Yes, the intention was indeed to use with items able to capture tabbed focus e.g. buttons elements.
